pennylane_qulacs/qulacs_device.py

```diff
@@ -295,12 +295,44 @@ def analytic_probability(self, wires=None):
         prob = self.marginal_prob(all_probs, wires)
         return prob
 
+    def expval(self, observable):
+        observable_map = {
+            "PauliX": "X",
+            "PauliY": "Y",
+            "PauliZ": "Z",
+            "Identity": "I",
+            "Hadamard": None,
+            "Hermitian": None,
+        }
+
+        if self.analytic:
+            qulacs_observable = Observable(self.num_wires)
+            if isinstance(observable.name, list):
+                observables = [observable_map[obs] for obs in observable.name]
+            else:
+                observables = [observable_map[observable.name]]
+
+            if None not in observables:
```

Comment:
Just to understand, could one not use the native methods for all observables that are not None here?

Reply:
I'm actually not sure how that would be done to be honest. This was simply the easiest way to implement it, and I think should, in most cases, be fine.